tools/fix_google_doc.php

```diff
@@ -270,3 +273,30 @@ function RunCmdStdinStdout($cmd, &$text) {
   fclose($pipes[1]);
   return proc_close($process);
 }
+
+// Handle situation when empty <a> tag was inserted before tag with content.
```

Comment:
// Move id attribute from stand-alone &lt;a> tag to the following sibling &lt;p> tag.
// It is necessary because tidy drops &lt;a> tags without content which are used as
// bookmarks in Google Docs.